### PR TITLE
feat(core): change default size for icon

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/core/components/button/button.style.less [hunk 1]
```diff
@@ -119,11 +119,19 @@
...
         justify-content: center;
     }
+
+    :host[tuiIconButton][data-size='s'] &,
+    :host[tuiIconButton][data-size='xs'] &,
+    :host[tuiIconButton][data-size='l'] &,
+    :host[tuiIconButton][data-size='m'] & {
+        width: 100%;
+        height: 100%;
+        max-width: 1.5rem;
```
Comment (waterplea) [inline projects/core/components/button/button.style.less:132]: Icons should be 1rem for `s` and `xs` sizes for both `tuiButton` and `tuiIconButton`.

Comment (splincode) [reply projects/core/components/button/button.style.less:132]: I have a problem then
image
image


I think it's breaking changes

**Comment by splincode** [comment]
image

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/card/examples/3/index.less [hunk 2]
```diff
@@ -44,5 +44,5 @@
     background-color: #7c48c3;
     background-image: linear-gradient(45deg, #c86dd7 0%, #3023ae 100%);
     margin-bottom: 1rem;
-    animation: spinCard 5s infinite;
+    //animation: spinCard 5s infinite;
```
Comment (splincode) [inline projects/demo/src/modules/components/card/examples/3/index.less:47]: ?

File: projects/kit/components/marker-icon/marker-icon.style.less [hunk 3]
```diff
@@ -167,26 +167,26 @@
...
 
 .t-icon {
-    width: 3rem;
-    height: 3rem;
+    width: 1.5rem;
+    height: 1.5rem;
 
     :host[data-size='xs'] & {
-        transform: scale(0.667);
+        transform: scale(1);
```
Comment (vladimirpotekhin) [inline projects/kit/components/marker-icon/marker-icon.style.less:174]: nit: do we need scale(1) ?

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**